### ISSUE TITLE
next version crash: Error: Couldn't find any versions for "@theia/languages" that matches "0.4.0-next.34e35a90"

### ISSUE THREAD

**Issue by avelino** [theia-ide/theia#4303]
```
yarn install v1.9.4
info No lockfile found.
[1/4] Resolving packages...
warning @theia/callhierarchy > @theia/core > inversify@4.14.0: Re-Released as 5.0.1
warning @theia/callhierarchy > @theia/core > electron > electron-download > nugget > progress-stream > through2 > xtend > object-keys@0.4.0:
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error Couldn't find any versions for "@theia/languages" that matches "0.4.0-next.34e35a90"
Error: Couldn't find any versions for "@theia/languages" that matches "0.4.0-next.34e35a90"
    at MessageError.ExtendableBuiltin (/opt/yarn/lib/cli.js:243:66)
    at new MessageError (/opt/yarn/lib/cli.js:272:123)
    at Function.<anonymous> (/opt/yarn/lib/cli.js:51264:13)
    at Generator.next (<anonymous>)
    at step (/opt/yarn/lib/cli.js:92:30)
    at /opt/yarn/lib/cli.js:103:13
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
Error: Couldn't find any versions for "@theia/monaco" that matches "0.4.0-next.34e35a90"
    at MessageError.ExtendableBuiltin (/opt/yarn/lib/cli.js:243:66)
    at new MessageError (/opt/yarn/lib/cli.js:272:123)
    at Function.<anonymous> (/opt/yarn/lib/cli.js:51264:13)
    at Generator.next (<anonymous>)
    at step (/opt/yarn/lib/cli.js:92:30)
    at /opt/yarn/lib/cli.js:103:13
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
The command '/bin/sh -c yarn --cache-folder ./ycache && rm -rf ./ycache' returned a non-zero code: 1
```

**Comment by vince-fugnitto** [comment]
Hi @avelino, can you perhaps share your `package.json` file so that I may see if there are any irregularities?

**Comment by avelino** [comment]
```
{
    "private": true,
    "dependencies": {
        "typescript": "latest",
        "@theia/callhierarchy": "next",
        "@theia/core": "next",
        "@theia/cpp": "next",
        "@theia/docker": "next",
        "@theia/editor": "next",
        "@theia/editorconfig": "next",
        "@theia/editor-preview": "next",
        "@theia/extension-manager": "next",
        "@theia/file-search": "next",
        "@theia/filesystem": "next",
        "@theia/git": "next",
        "@theia/go": "next",
        "@theia/getting-started": "next",
        "@theia/java": "next",
        "@theia/json": "next",
        "@theia/keymaps": "next",
        "@theia/languages": "next",
        "@theia/markers": "next",
        "@theia/merge-conflicts": "next",
        "@theia/messages": "next",
        "@theia/metrics": "next",
        "@theia/mini-browser": "next",
        "@theia/monaco": "next",
        "@theia/navigator": "next",
        "@theia/outline-view": "next",
        "@theia/php": "next",
        "@theia/output": "next",
        "@theia/plantuml": "next",
        "@theia/plugin": "next",
        "@theia/plugin-ext": "next",
        "@theia/plugin-ext-vscode": "next",
        "@theia/preferences": "next",
        "@theia/preview": "next",
        "@theia/process": "next",
        "@theia/python": "next",
        "@theia/ruby": "next",
        "@theia/rust": "next",
        "@theia/search-in-workspace": "next",
        "@theia/task": "next",
        "@theia/terminal": "next",
        "@theia/textmate-grammars": "next",
        "@theia/tslint": "next",
        "@theia/typescript": "next",
        "@theia/userstorage": "next",
        "@theia/variable-resolver": "next",
        "@theia/workspace": "next",
        "theia-yang-extension": "next",
        "@theia/yaml": "0.1.0"
    },
    "devDependencies": {
        "@theia/cli": "next"
    }
}

```

**Comment by kittaakos** [comment]
@avelino, I tried to install the dependencies with the `package.json` you provided; it worked. 

```bash
Akoss-MacBook-Pro:tmp-theia akos.kitta$ yarn
yarn install v1.12.3
info No lockfile found.
[1/4] 🔍  Resolving packages...
warning @theia/callhierarchy > @theia/core > inversify@4.14.0: Re-Released as 5.0.1
warning @theia/callhierarchy > @theia/core > electron > electron-download > nugget > progress-stream > through2 > xtend > object-keys@0.4.0: 
warning @theia/callhierarchy > @theia/monaco > @theia/filesystem > trash > xdg-trashdir > @sindresorhus/df > execa > cross-spawn-async@2.2.5: cross-spawn no longer requires a build toolchain, use it instead
warning @theia/core > inversify@4.14.0: Re-Released as 5.0.1
warning @theia/docker > @theia/core > inversify@4.14.0: Re-Released as 5.0.1
warning @theia/extension-manager > @theia/application-manager > less > request > hawk > boom@2.10.1: This version is no longer maintained. Please upgrade to the latest version.
warning @theia/extension-manager > @theia/application-manager > less > request > hawk > cryptiles@2.0.5: This version is no longer maintained. Please upgrade to the latest version.
warning @theia/extension-manager > @theia/application-manager > less > request > hawk > cryptiles > boom@2.10.1: This version is no longer maintained. Please upgrade to the latest version.
warning @theia/extension-manager > @theia/application-manager > less > request > hawk > hoek@2.16.3: This version is no longer maintained. Please upgrade to the latest version.
warning @theia/extension-manager > @theia/application-manager > less > request > hawk > boom > hoek@2.16.3: This version is no longer maintained. Please upgrade to the latest version.
warning @theia/extension-manager > @theia/application-manager > less > request > hawk > sntp > hoek@2.16.3: This version is no longer maintained. Please upgrade to the latest version.
warning @theia/extension-manager > @theia/application-manager > webpack-cli > jscodeshift > babel-preset-es2015@6.24.1: 🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read babeljs.io/env to update! 
warning @theia/extension-manager > @theia/application-manager > webpack-cli > webpack-addons > jscodeshift > babel-preset-es2015@6.24.1: 🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read babeljs.io/env to update! 
warning @theia/extension-manager > @theia/application-manager > webpack-cli > jscodeshift > nomnom@1.8.1: Package no longer supported. Contact support@npmjs.com for more info.
warning @theia/extension-manager > @theia/application-manager > webpack-cli > webpack-addons > jscodeshift > nomnom@1.8.1: Package no longer supported. Contact support@npmjs.com for more info.
warning @theia/extension-manager > @theia/application-manager > css-loader > cssnano > autoprefixer > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @theia/extension-manager > @theia/application-manager > css-loader > cssnano > postcss-merge-rules > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @theia/extension-manager > @theia/application-manager > css-loader > cssnano > postcss-merge-rules > caniuse-api > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @theia/go > @theia/core > inversify@4.14.0: Re-Released as 5.0.1
warning @theia/go > go-language-server > json-rpc2 > es5class@2.3.1: this package isn't maintained anymore because ES6+
warning @theia/php > @theia/core > inversify@4.14.0: Re-Released as 5.0.1
warning @theia/plantuml > @theia/core > inversify@4.14.0: Re-Released as 5.0.1
warning @theia/ruby > @theia/core > inversify@4.14.0: Re-Released as 5.0.1
warning @theia/rust > @theia/core > inversify@4.14.0: Re-Released as 5.0.1
warning theia-yang-extension > @theia/core > inversify@4.14.0: Re-Released as 5.0.1
warning theia-yang-extension > theia-sprotty > @theia/core > inversify@4.14.0: Re-Released as 5.0.1
warning theia-yang-extension > yang-sprotty > sprotty > inversify@4.14.0: Re-Released as 5.0.1
warning @theia/yaml > @theia/core > inversify@4.14.0: Re-Released as 5.0.1
[2/4] 🚚  Fetching packages...
warning monaco-languageclient@0.9.0: The engine "vscode" appears to be invalid.
warning vscode-base-languageclient@4.4.0: The engine "vscode" appears to be invalid.
warning go-language-server@0.1.7: The engine "vscode" appears to be invalid.
warning vscode-extension-telemetry@0.0.10: The engine "vscode" appears to be invalid.
warning vscode-languageclient@2.6.3: The engine "vscode" appears to be invalid.
[3/4] 🔗  Linking dependencies...
warning "@theia/extension-manager > @theia/application-manager > font-awesome-webpack@0.0.5-beta.2" has unmet peer dependency "font-awesome@>=4.3.0".
warning "@theia/tslint > tslint-language-service@0.9.9" has incorrect peer dependency "typescript@>= 2.3.1 < 3".
warning "@theia/tslint > tslint-language-service@0.9.9" has unmet peer dependency "tslint@>= 4 < 6".
warning "theia-yang-extension > yang-sprotty > sprotty > snabbdom-virtualize@0.7.0" has incorrect peer dependency "snabbdom@~0.6.6".
[4/4] 📃  Building fresh packages...
success Saved lockfile.
✨  Done in 184.76s.
```

[`@theia/languages@0.4.0-next.34e35a90` has been successfully published](https://www.npmjs.com/package/@theia/languages/v/0.4.0-next.34e35a90) to npmjs.org, perhaps, you were unlucky and ran `yarn` when the CI server was publishing the extensions and somehow you ended up having different `next` versions.

You can try the followings:
 - Run `yarn` again.
 - Run `yarn --force`.
 - If it still does not help, you can wipe `yarn.lock` and run `yarn` again.
 - Still does not work, let us know.

I hope this helps.

**Comment by avelino** [comment]
thanks, used  `--force` it worked
